app/controllers/contacts_controller.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class ContactsController < ApplicationController
+  PERSON_FIELDS_EDIT_REQUESTABLE = [:name, :country_iso2, :gender, :dob].freeze
```

Comment:
This should probably(? feel free to argue) be a part of `person.rb`, because that's the model you're actually trying to edit